server.js

```diff
@@ -1,29 +1,68 @@
-const http = require("http") 
-const fs = require("fs/promises")
+const http = require("http");
+const fs = require("fs/promises");
+const cache = require("node-cache");
 
-const getFiles = async ()=>{
-    const indexHtml = await fs.readFile("./index.html")
-    const indexJs = await fs.readFile("./dist/r.bundle.js")
-    return [indexHtml,indexJs]
+const storeCache = new cache();
+
+if (!storeCache.has("registrations")) {
+    storeCache.set("registrations", []);
```

Comment:
Is there any specific reason why you decided to use cache instead of the plain variables? One of the main advantages of cache is the possibility to set expiration time, getting data by keys etc. What you've done here is not different from using const registrations = []. If you want to use cache properly instead of using array, set every user as separately. 
For example, if you have mail: test@gmail.com, and password: pass124, you should use cache like this: `storeCache.set(mail, password, <some expiration time, 5 minutes will be fine in this case>)`

Reply:
Later on instead of looping over all records you can easily check if email and password already exist: 
`storeCache.has(mail)` <- check if mail exist in cache
and then if mail exist:
`storeCache.get(mail) === password`  <- check if password exist in cache

Reply:
Do not guess. Read the documentation carefully and if you understand it, try cache approach. Otherwise, just use a simple variable `const registrations = []`